src/main/java/com/dani/blog/domain/Comment.java

```diff
@@ -0,0 +1,17 @@
+package com.dani.blog.domain;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * @author dani
+ */
+@NoArgsConstructor
+@Getter
```

Comment:
Recommending `@Data` annotation, as it implements all these +getHash, equals, toString.